graphics/graphviz/Portfile

```diff
@@ -30,17 +22,23 @@ long_description                Graphviz is ${description}. Graph visualization
                                 simple text language, and produce output in a \
                                 variety of visual and text formats.
 
-checksums                       rmd160  8fc103fc519e07e9db522ec2f748838571c71acf \
-                                sha256  ca5218fade0204d59947126c38439f432853543b0818d9d728c589dfe7f3a421
+use_parallel_build              yes
+use_bzip2                       no
+use_xz                          yes
```

Comment:
You can eliminate both `use_bzip2` and `use_xz`; the gitlab portgroup takes care of it.